tests/redux/reducers/search/searchFiltersReducer-test.js

```diff
@@ -630,6 +629,39 @@ describe('searchFiltersReducer', () => {
         });
     });
 
+    describe('UPDATE_PRICING_TYPE', () => {
```

Comment:
The `contractFilterFunctions` add additional functionality besides simply creating a Set from a single value. Can you expand your tests to cover what happens if a value already exists (it should delete) or you try to add multiple values, some of which may already exist (it should append and/or delete)?